$NetBSD: patch-ab,v 1.1.1.1 2005/06/25 01:55:09 reed Exp $

the font descriptor strings have * in them, they need to be quoted

--- icewm4vim/icewm.vim.orig	2005-03-13 13:21:06.000000000 +0000
+++ icewm4vim/icewm.vim	2005-03-13 13:21:28.000000000 +0000
@@ -20,7 +20,7 @@
 	if stridx(font, "Font") != -1
 		let font = substitute(font, "\"", "", "g")
 		let font = substitute(font, ".*=", "", "")
-		let font = system("icefonsel " . font)
+		let font = system("icefonsel \"" . font . "\"")
 		if font != ""
 			exe "s/=.*/".font
 		endif
